React/wbooks/src/app/screens/Login/components/LoginForm/index.js

```diff
@@ -1,69 +1,71 @@
 import React, { Component } from 'react';
-import { Redirect } from 'react-router';
-import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
 
-import AccessError from '../../../../components/AccessError/index.js';
-import userIcon from '../../../../assets/user_icon.png';
-import passwordIcon from '../../../../assets/password.png';
 import * as regexs from '../../../../../utils/regexs';
-import * as authService from '../../../../../services/authService';
+import * as sessionActions from '../../../../../redux/session/actions';
+import * as booksActions from '../../../../../redux/books/actions';
 
-import styles from './styles.scss';
+import LoginForm from './layout';
 
-class LoginForm extends Component {
-  state = { name: '', password: '' , hasErrors: '', isLogged: false };
+class LoginFormContainer extends Component {
+  handleSubmit = async () => {
+    const user = await this.props.handleSubmit(this.props.userName, this.props.password);
+    this.props.loadBooks(user);
```

Comment:
🤔  ammm this should not be here. you must dispatch this action in the `componentWillMount` of `Home`, and delete the `async` `await`